### PR TITLE
Dependencies tab

### PR DESCRIPTION
**Pull request by KutnerUri**
## Proposed Changes

- Add Graph section to component page
- Refactor sidebar widgets to be reused in graph.

### CODE REVIEW

File: extensions/graph/ui/query/node-model.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { RawNode } from './get-graph.query';
+
+export class NodeModel {
+  id: string;
+  component: ComponentModel;
+
+  static from(rawNode: RawNode) {
+    const node = new NodeModel();
+    node.id = rawNode.id;
+    // @TODO - component model should not expect all fields to have values
```
Comment (odedre) [inline extensions/graph/ui/query/node-model.ts:11]: I noticed you changed the componentModel fields to optional anyway

**Review by odedre — APPROVED** [review]

File: components/env-icon/env-icon.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ComponentModel } from '@teambit/component';
+
+export type EnvIconProps = { component: ComponentModel } & React.HTMLAttributes<HTMLDivElement>;
```
Comment (GiladShoham) [inline components/env-icon/env-icon.tsx:4]: why div?

File: extensions/graph/graph.ui.runtime.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  constructor(private componentWidgetSlot: ComponentWidgetSlot) {}
+  static dependencies = [ComponentAspect];
+  static runtime = UIRuntime;
+  static slots = [Slot.withType<ComponentWidget>()];
+  static async provider([componentUI]: [ComponentUI], config, [componentWidgetSlot]: [ComponentWidgetSlot]) {
+    const graphUI = new GraphUI(componentWidgetSlot);
+
+    // TODO - remove feature flag when data provider works for all workspaces
+    if (window.location.pathname !== '/enable-graph') return;
```
Comment (GiladShoham) [inline extensions/graph/graph.ui.runtime.tsx:35]: why pathname and not query param?

Comment (KutnerUri) [reply extensions/graph/graph.ui.runtime.tsx]: it's easier to got to localhost:3000/enable-graph as a landing page

**Pull request merged**